app/models/appointment.rb

```diff
@@ -27,7 +27,7 @@ class Appointment < ApplicationRecord
   validates :status, presence: true
   validates :guider, presence: true
 
-  validate :not_within_two_business_days
+  validate :not_within_two_business_days, unless: :agent_is_resource_manager?
```

Comment:
If the user is a guider (or any other type of user) then we need to validate that the appointment starts two business days from now.